### DISCUSSION TITLE
Bug: syntax error on "about"

### DISCUSSION THREAD

**Discussion by ericwb** [community/community#4383] [Projects and Issues] [answered]
If I use the key "about" in the yaml issue template (which worked in the markdown version), I get error:

`about is not a permitted key. Learn more. `

However, if I remove "about", I also get this error:

```
There is a problem with this template
About can't be blank. 
```

The only way to resolve this is to use the key "description", not "About" as described in the error.  So definitely a bug in the
parsing and error reporting of the yaml format.

**Comment by lukehefson** [comment, marked as answer]
Hey @ericwb! This is actually intended behaviour and not a bug.

We’re currently in the process of centralising on number of ‘code as config’ syntax choices across GitHub – and as we move forward with YAML issue forms as a _new_ product – we wanted to reflect this new direction and implement `description` on favour of `about`.

**Reply by ericwb** [reply]
Then there is still a bug in there if `description` is to be used instead of `about`  The error message should read instead:

```
There is a problem with this template
Description can't be blank. 
```
